### DISCUSSION TITLE
Drag and Drop pencil lines

### DISCUSSION THREAD

**Discussion by AA757200** [MoffKalast/Sailwind-Map#17] [Ideas]
Is it possible to use the pencil tool to draw a line, then click and drag that to somewhere else on the map while maintaining the length and bearing of the drawn line? This would make running fixes a possibility.

**Comment by MoffKalast** [comment]
Hmm I think it would be possible to implement, but probably better as another tool to avoid misclicks altering existing drawings.

A while back I've looked at how it might be doable to make circles with a fixed NM radius, which would be the more legit option for this, but that is actually problematic with arcgis since circles are a finite precision polygon and that won't be super accurate once zoomed in... which is exactly what people would be doing when looking for overlaps lol.
